src/app/server/BUILD.gn

```diff
@@ -15,12 +15,21 @@
 import("//build_overrides/chip.gni")
 import("${chip_root}/src/app/common_flags.gni")
 
+# Move this inside the GroupDataProvider once implemented Issue #11075
+declare_args() {
+  chip_enable_test_group = false
```

Comment:
Could you add a comment or maybe even expand what "test_group" does a bit more? Reading without 'multicast' context tells me "this groups testing" which is probably wrong.

how about `chip_enable_group_messaging_tests` or something similar. Same with the define below,  a name of `CHIP_ENABLE_GROUP_MESSAGING_TESTS` seems to be easier to read.

Reply:
Will do in a follow up PR

Reply:
Done